tests/test_units/test_rms_context.py

```diff
@@ -554,6 +554,32 @@ def test_gridproperty_export_file_set_name(rmssetup, rmsglobalconfig, gridproper
     )
 
 
+@inside_rms
+def test_gridproperty_export_with_geometry(
+    rmssetup, rmsglobalconfig, grid, gridproperty
+):
+    """Export the the grid and the gridprop(s) with connected geometry."""
+    logger.info("Active folder is %s", rmssetup)
+    os.chdir(rmssetup)
+
+    edata = dataio.ExportData(
+        config=rmsglobalconfig, content="depth"
```

Comment:
```suggestion
        config=rmsglobalconfig, content="cpgrid"
```

Reply:
I don't find that `cpgrid` is a valid content...

Reply:
Completely True 😅 I mixed up with the class. Then disregard my comment here!